frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -110,15 +119,11 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
 
       const { value } = event.currentTarget as HTMLInputElement;
       setSearchQuery(value || '');
-      props.actions.fetchOperations({
-        limit: props.limit,
-        offset: 0,
-        search: value || '',
-        mine: showMyQueries,
-        link: props.sourceID
-          ? getSourceDatasetsLink(props.sourceID, showMyQueries, props.limit, 0, value)
-          : undefined,
-      });
+      setPageNumber(1);
+      const values = queryString.parse(location.search);
+      values.q = value || null;
+      values.page = '1';
+      props.history.push(`${window.location.pathname}?${queryString.stringify(values)}`);
```

Comment:
I see this pattern a few times ... page number, values, history push ... sure you can extract it into a reusable function